src/main/java/parking/controllers/DriverController.java

```diff
@@ -0,0 +1,93 @@
+package parking.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import parking.entities.Plate;
+import parking.entities.enums.Region;
+import parking.services.PlateManagerService;
+import parking.services.PricingService;
+
+import javax.validation.Valid;
+import java.rmi.activation.UnknownObjectException;
```

Comment:
Czemu korzystasz z tej klasy? Co to jest RMI?

Reply:
j/w. Problem leżał w tym, że chciałem pokryć sytuację, w której użyto wartości nie znajdującej się enumie Region. W sumie InvalidArgumentException by się lepiej nadawał.
RMI - Remote Method Invocation.

Reply:
Poprawka zamiast InvalidArgumentException IllegalArgumentException